CLDConfig/py_utils.py

```diff
@@ -228,3 +228,16 @@ def parse_collection_patch_file(patch_file: Union[str, os.PathLike]) -> List[str
 
     # Flatten the list of lists into one large list
     return [s for strings in patch_colls for s in strings]
+
+
+def toMarlinDict(dict: Dict[str, Any]) -> Dict[str, Any]:
+    """Convert a dictionary to a Marlin-compatible format.
+
+    Args:
+        dict (Dict[str, Any]): The dictionary to convert.
+
+    Returns:
+        Dict[str, Any]: The converted dictionary.
+    """
+
+    return {k: [str(v).lower()] if isinstance(v, bool) else [str(elem) for elem in v] if isinstance(v, list) else [str(v)] if isinstance(v, float) or isinstance(v, int) else [v] if isinstance(v, str) else v for k, v in dict.items()}
```

Comment:
```suggestion
    return {k: [str(v).lower()] if isinstance(v, bool) else [str(elem) for elem in v] if isinstance(v, list) else [str(v)] if isinstance(v, (float, int)) else [v] if isinstance(v, str) else v for k, v in dict.items()}
```
isinstance can take a tuple of types as the second argument
Or put the `str` also in that tuple for simplicity.